dallinger/data.py

```diff
@@ -16,17 +17,19 @@
 import postgres_copy
 import psycopg2
 
-try:
-    import odo
-    import pandas as pd
-    import tablib
-except ImportError:
-    pass
-
 from dallinger import heroku
 from dallinger import db
 from dallinger import models
 
+with warnings.catch_warnings():
+    warnings.simplefilter(action='ignore', category=FutureWarning)
+    try:
+        import odo
+        import pandas as pd
+        import tablib
+    except ImportError:
+        pass
```

Comment:
I recall having errors when looping through my IEC algorithm because my virtual environment did not have a few of these imports. But because the `ImportError` was passed, it wasn't immediately obvious that they were missing without examining this piece. Logging this as `debug` or `info` would help unmask that confusion .